Sources/AppcuesKit/Presentation/Public/AppcuesViewElement.swift

```diff
@@ -74,12 +76,13 @@ internal extension AppcuesViewElement {
     // Used internally by the SDK to find all views, recursively, that have at least a
     // partial match with the given selector. Used to find the position of an element
     // targeted experience.
-    func viewsMatchingSelector(_ target: AppcuesElementSelector) -> [AppcuesViewElement] {
-        var views: [AppcuesViewElement] = []
+    func viewsMatchingSelector(_ target: AppcuesElementSelector) -> [(AppcuesViewElement, Int)] {
```

Comment:
My personal preference is to name secondary, ambiguous (eg "what's the Int at first glance?") tuple elements, so I'd say
`func viewsMatchingSelector(_ target: AppcuesElementSelector) -> [(AppcuesViewElement, weight: Int)] {` is clearer.

The name doesn't have to be used anywhere else (though it can be used at the call site for additional clarity), but it makes it a bit clearer to read the intent.

Could do `(view: AppcuesViewElement, weight: Int)` too if we really want `AppcuesTargetElementTrait` to be more readable.

Reply:
good call, updating to use `(view: AppcuesViewElement, weight: Int)` and usage in `AppcuesTargetElementTrait` to maximize readability.